src/screens/Embed/Embed.js

```diff
@@ -58,28 +58,21 @@ export default function Embed() {
   }, [_location, countyId, countyFipsId]);
 
   const projections = useProjections(location, selectedCounty);
-  const stateSummaryData = useStateSummaryData(location);
   const locationName = STATES[location];
   const intervention = STATE_TO_INTERVENTION[location];
-  let summaryData = stateSummaryData;
-  if (stateSummaryData && selectedCounty) {
-    summaryData = _.find(summaryData.counties, [
-      'fips',
-      selectedCounty.full_fips_code,
-    ]);
-  }
-
-  if (!projections || !summaryData) {
+  if (!projections) {
     return null;
   }
 
-  if (!projections.primary) {
+  const primary = projections.primary;
+  if (!primary) {
     return <span>'No data available for county.'</span>;
   }
 
-  const { cases, deaths } = summaryData;
+  const cases = primary.currentCumulativeCases;
+  const deaths = primary.currentCumulativeDeaths;
 
-  const totalPopulation = projections.baseline.totalPopulation;
+  const totalPopulation = primary.totalPopulation;
```

Comment:
projections.baseline is the NO_INTERVENTION endpoint whereas primary will be OBSERVED_INTERVENTION, but it doesn't matter which one we use. I'm just using `primary` to be consistent.